### DISCUSSION TITLE
Смена положения плашки

### DISCUSSION THREAD

**Discussion by Cerefka** [ilyhalight/voice-over-translation#567] [Q&A] [answered]
Друзья, я не кодер, поэтому прошу помощи. Как то месяцы назад запарился и переместил кнопку перевода с центра видео в верхний левый угол (при помощи chatGPT) и сделал её полупрозрачной. Было очень удобно и она не раздражала меня находясь в центре. Но сейчас переустановив скрипт, всё вернулось к исходным настройкам, а найти параметры заново не получается. Пожалуйста подскажите какие параметры в коде отвечают за положение и прозрачность.

**Comment by ilyhalight** [comment, marked as answer]
Держите готовый юзерстайл скрипт, чтобы не менять значения при обновление ([гайд по установке ](https://github.com/ilyhalight/voice-over-translation?tab=readme-ov-file#%D0%BA%D0%B0%D1%81%D1%82%D0%BE%D0%BC%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D1%8F-%D0%B2%D0%BD%D0%B5%D1%88%D0%BD%D0%B5%D0%B3%D0%BE-%D0%B2%D0%B8%D0%B4%D0%B0)на главной). Прозрачность задается из кода, поэтому ее нужно будет менять при каждом обновление. Для ее смены ищите в расширение, где вызывается функция logout и меняйте на нужную вам прозрачность. 

```
/* ==UserStyle==
@name         vot-btn-left-top
@version      02.04.2024
@namespace    vot-btn-left-top
@description  LLL
@author       Toil
@license      No License
==/UserStyle== */

.vot-segmented-button, .vot-menu {
    left: 10%;
}
```

**Reply by Cerefka** [reply]
Спасибо большое. Я в итоге разобрался благодаря вашей подсказке. И даже удалось сделать всё меню прозрачным. Делал методом тыка, но всё сработало через этот код

```
/* ==UserStyle==
@name         vot-btn-left-top
@version      02.04.2024
@namespace    vot-btn-left-top
@description  LLL
@author       Toil
@license      No License
==/UserStyle== */
.vot-menu {
    left: 17.5rem;
    top: 7rem;
    background: rgba(0, 0, 0, .5)
}

.vot-segmented-button {
    left: 13rem;
    top: 2rem;
    background: rgba(0, 0, 0, .5)
}

.vot-dialog {
    background: rgba(0, 0, 0, .8)
}
```
